privacy-snapshot.json

```diff
@@ -58,5 +58,6 @@
   "oidc.api.cx.metamask.io",
   "price.api.cx.metamask.io",
   "token.api.cx.metamask.io",
-  "client-side-detection.api.cx.metamask.io"
+  "client-side-detection.api.cx.metamask.io",
+  "user-storage.api.cx.metamask.io"
```

Comment:
This API is now introduced in e2e tests as we have launched a new "Account Syncing" feature, which will sync your accounts you may have previously stored. This is done on wallet start/open post onboarding.